pkg/api/proto/runme/reporter/v1/reporter.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+package runme.reporter.v1;
+
+import "runme/parser/v1/parser.proto";
+
+option go_package = "github.com/stateful/runme/v3/pkg/api/gen/proto/go/runme/reporter/v1;reporterv1";
+
+message TransformRequest {
+  runme.parser.v1.Notebook notebook = 1;
+  optional bool auto_save = 2;
```

Comment:
Ln 11-30, we should group into one or multiple sub-messages. Protos are practically immutable and this laundry list of properties at the request message level is likely a long-term liability. `Host`, `Env`, `VSC`? Honestly not even as important how it's grouped but it needs grouping.

Reply:
Got it. I structured it this way so the "Transform" function can return the payload in the shape defined by the proto file, eliminating the need to do this on the client side. But I totally get your point 👍

Reply:
Right. Perhaps group them under a single message that's part of `TransformRequest`.

Reply:
Addresed with a single "extension" message.